blade-graphics/src/gles/command.rs

```diff
@@ -227,6 +227,24 @@ impl super::PassEncoder<'_, super::RenderPipeline> {
     ) -> super::PipelineEncoder<'b> {
         self.commands
             .push(super::Command::SetProgram(pipeline.inner.program));
+
+        match &pipeline.inner.blend_states[..] {
+            &[single] => self.commands.push(super::Command::SetBlendState(single)),
+            separate => self
+                .commands
+                .extend(separate.iter().zip(0..).map(|(&blend_state, i)| {
+                    super::Command::SetBlendStatePerDrawBuffer(i, blend_state)
+                })),
+        }
+        match &pipeline.inner.write_masks[..] {
```

Comment:
this could also look less fancy if we treated them together, something like:
```rust
if pipeline.inner.target_states.len() == 1 {
  let state = &pipeline.inner.target_states[0];
   self.commands.push(super::Command::SetBlendState(state.blend));
   self.commands.push(super::Command::SetWriteMask(state.write_mask));
} else {
  for state in pipeline.inner.target_states.iter() {
    ...
  }
}
```
We could also just merge the `Command` variant itself, since it would be used together all the time, further simplifying the whole thing